### ISSUE TITLE
Bad scope given to the notifyChange function

### ISSUE THREAD

**Issue by elias75015** [iTowns/itowns#1108]
In some cases, when bad requests are launched by the iTowns client, the notifyChange function throws an error.

I don't really know why, but in this case, the scope is not correctly set : "Window" is assigned to "this", and then the "_changeSources" property is undefined :

```
  if (changeSource) { 
    this._changeSources.add(changeSource);
  }
```

The error : 
> View.js:385 Uncaught TypeError: Cannot read property 'add' of undefined
    at Mt.notifyChange (View.js:385)

Maybe the context should be binded to the notifyChange function when called, here for example (LayerMaterialNodeProcessing.js) :

 `
window.setTimeout(context.view.notifyChange, node.layerUpdateState[layer.id].secondsUntilNextTry() * 1000, node, false);`

**Comment by elias75015** [comment]
Hello,

any news about this one ? Tell me if you need more precision.

Thank you !

**Comment by gchoqueux** [comment]
I'll fix the issue today

**Comment by gchoqueux** [comment]
it would have been fixed in 2.9.0. Could you try this version?

**Comment by elias75015** [comment]
It seems OK with the 2.9 indeed : Thanks
